dockertest/subtestbase_unittests.py

```diff
@@ -72,6 +74,36 @@ class DockerTestFail(Exception):
 ]
 
 
+class TestNotRedHat(TestCase):
+
+    def setUp(self):
+        import subtestbase
+        self.subtestbase = subtestbase
+        # Saves some typing
+        self.stbsb = self.subtestbase.SubBase
+        pfx = 'subtestbase_unitest'
+        (self.fd,
```

Comment:
Why preserve fd? Making this `None` saves maintainer effort, and also lets you remove the `os.close()` in `tearDown`.